From 1518beca0b5a2730a87da9ce82299ef6469bfe8a Mon Sep 17 00:00:00 2001
From: "Anthony G. Basile" <blueness@gentoo.org>
Date: Fri, 23 Aug 2019 11:29:06 -0400
Subject: [PATCH] consoletype.c: update header to <sys/sysmacros.h>

This is an update to commit 53b60b4.  Recent changes in musl have
removed the inclusion of <sys/sysmacros.h> from <sys/types.h>.
Now the macros major()/minor() require the inclusion of the former
header.  This change does not affect glibc nor earlier versions of
musl, but is required for musl 1.1.23 and later.

Signed-off-by: Anthony G. Basile <blueness@gentoo.org>
---
 consoletype.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/consoletype.c b/consoletype.c
index 0eb7a07..ece0d5f 100644
--- a/consoletype.c
+++ b/consoletype.c
@@ -13,7 +13,7 @@
 #include <string.h>
 #include <sys/ioctl.h>
 #include <sys/stat.h>
-#include <sys/types.h>
+#include <sys/sysmacros.h>
 #include "headers.h"
 
 enum termtype {
-- 
2.21.0

